cli/config/configuration-fc.toml.in

```diff
@@ -289,3 +289,9 @@ disable_guest_seccomp=@DEFDISABLEGUESTSECCOMP@
 # If you are using docker, `disable_new_netns` only works with `docker run --net=none`
 # (default: false)
 #disable_new_netns = true
+
+# If enabled, the runtime will enable experimental features.
+# At this time, experimental features are not stable enough for production.
+# These features will break compatibility, and are prepared for a big version bump.
```

Comment:
Should this be "may break" instead of "will break"? We may have an experimental feature which is being tested out in parts but may not be breaking compatibility.

Reply:
Sounds good